PROJECT.md

```diff
@@ -0,0 +1,16 @@
+# Project Guidelines
+
+Each Project repository should have, at a minimum:
+
+1. Contributing Guidelines, we use a [minimal template](#minimal-contributing-guidelines) that points to our main Contributing Guidelines and can be extended as needed to cover project-specific requirements.
+1. License, we use GPLv3 as our _preferred_ license for code and Creative Commons 4.0 BY-SA for creative works including documentation. It is **required** that all new repos are created with a license and copyright statement.
```

Comment:
It would be *great* to work up an example or copy+paste section to drop in a project’s README. I’m not sure the [“license & copyright” section of web-monitoring-db’s README](https://github.com/edgi-govdata-archiving/web-monitoring-db#license--copyright) is ideally worded and it would be good to have something standard.